PYME/recipes/processing.py

```diff
@@ -1959,3 +1959,103 @@ def execute(self, namespace):
             pass
             
         namespace[self.output_name] = res
+
+@register_module('DarkAndVarianceMap')
+class DarkAndVarianceMap(ModuleBase):
+    input = Input('input')
+    output_variance = Output('variance')
+    output_dark = Output('dark')
+    dark_threshold = Float(10000)  # this really should depend on the gain mode (12bit vs 16 bit etc)
+    variance_threshold = Float(90000)  # again this is currently picked fairly arbitrarily
+    start = Int(0)
+    end = Int(0)
+    uniform = Bool(False)  # simulate a uniform map
+
+    def execute(self, namespace):
+        from PYME.IO.MetaDataHandler import NestedClassMDHandler
+        from PYME.Analysis import gen_sCMOS_maps
+
+        logging.basicConfig() # without it got 'No handlers could be found for logger...'
```

Comment:
not safe here - will override any previous logging config. This could be a bug if `dh5view` doesn't setup logging.

Reply:
Actually I see it's copied from the `main()` function in gen_scmos_maps.py, it makes sense in the main function of a standalone program, but shouldn't be copied over to a library.